### PR TITLE
added resource for ssl certificate order

### PR DESCRIPTION
**Pull request by sagarkale1**
1. Added resource for requesting a ssl certificate.
2. Added documentation for requesting a ssl certificate.
3. Verified and Tested, It's working fine.

### CODE REVIEW

**Comment by vijaylakshmi-b** [comment]
@hkantare Could you please review this PR?

File: ibm/resource_ibm_ssl_certificate.go [hunk 1]
```diff
@@ -0,0 +1,830 @@
...
+
+			"billing_address_same_as_organization_flag": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"organization_information": {
+				Type:     schema.TypeSet,
+				Required: true,
```
Comment (hkantare) [inline ibm/resource_ibm_ssl_certificate.go]: Add MaxItems:      1, so it restricts to user to provide organization_information only once similarly for technical_contact, billing_contact, administrative_contact.....

File: ibm/resource_ibm_ssl_certificate.go [hunk 2]
```diff
@@ -0,0 +1,830 @@
...
+		if err != nil {
+			return fmt.Errorf("Error during creation of ssl: %s", err)
+		}
+
+		ssl, err := findSSLByOrderId(sess, *receipt.OrderId)
+		d.SetId(fmt.Sprintf("%d", *ssl.Id))
+		return resourceIBMSSLCertificateRead(d, m)
+	} else {
+		log.Println("Provided CSR is not valid.")
+		return nil
```
Comment (hkantare) [inline ibm/resource_ibm_ssl_certificate.go]: Instead returning nil return the error

File: website/docs/r/ssl_certificate.html.markdown [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+* `admin_city` - (Optional, string) The city for administrative contact.
+* `admin_postalCode` - (Optional, integer) The postal code for administrative contact.
+* `admin_state` - (Optional, string) The two letter state code of administrative contact. Allowed value for country which doesn't have states is `OT`.
+* `admin_countryCode` - (Optional, string) The two letter country code for administrative contact.
+* `admin_organizationName` - (Optional, string) Name of organization for administrative contact.
+* `admin_firstName` - (Optional, string) The first name for administrative contact.
+* `admin_lastName` - (Optional, string) The last name for administrative contact.
+* `admin_title` - (Optional, string) The title for for administrative contact.
+* `admin_emailAddress` -(Optional, string) email address for administrative contact.
+* `billing_addressLine1` - (Optional, string) The address for billing contact.
```
Comment (hkantare) [inline website/docs/r/ssl_certificate.html.markdown]: Add all high level attributes organization_information and define the inner attributes as nested attributes...for all attributes(techincal,administrative..so on)
https://ibm-cloud.github.io/tf-ibm-docs/v0.16.1/r/iam_user_policy.html

Comment (hkantare) [reply website/docs/r/ssl_certificate.html.markdown]: Add Attribute Reference section also

**Comment by hkantare** [comment]
General Remove unnecessary log staments..

File: ibm/resource_ibm_ssl_certificate.go (hunk 1 above)
Comment (sagarkale1) [reply ibm/resource_ibm_ssl_certificate.go]: Done.

File: website/docs/r/ssl_certificate.html.markdown (hunk 3 above)
Comment (sagarkale1) [reply website/docs/r/ssl_certificate.html.markdown]: Done. please have a look.

File: ibm/resource_ibm_ssl_certificate.go (hunk 2 above)
Comment (sagarkale1) [reply ibm/resource_ibm_ssl_certificate.go]: Done.

**Comment by sagarkale1** [comment]
@hkantare Made suggested changes please review.

**Comment by hkantare** [comment]
We have conflict in provider.go ..Please resolve the conflict .

**Comment by sagarkale1** [comment]
@hkantare Resolved merge conflicts.

**Pull request merged**